src/Symfony/Component/Serializer/Normalizer/PropertyNormalizer.php

```diff
@@ -68,23 +68,17 @@ public function normalize($object, $format = null, array $context = array())
             if (isset($this->callbacks[$property->name])) {
                 $attributeValue = call_user_func($this->callbacks[$property->name], $attributeValue);
             }
-            if (null !== $attributeValue && !is_scalar($attributeValue)) {
-                if (!$this->serializer instanceof NormalizerInterface) {
-                    throw new LogicException(sprintf('Cannot normalize attribute "%s" because injected serializer is not a normalizer', $property->name));
-                }
-
-                $attributeValue = $this->serializer->normalize($attributeValue, $format, $context);
-            }
 
-            $propertyName = $property->name;
-            if ($this->nameConverter) {
-                $propertyName = $this->nameConverter->normalize($propertyName);
+            // Recursive call are done at the end of the process to allow @MaxDepth to work
```

Comment:
`Recursive call are done` -> `Recursive call is done` or `Recursive calls are done`